draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -493,6 +493,8 @@ and sent over clear text.
 To minimize impact of active on-path attacks on the DNS channel, the
 client validates the response as described in {{client-processing}}.
 
+If the client can display an error page to the end-user, it MUST generate the error page. The "c," "j," and "o" fields MUST NOT be used to display an error page.
```

Comment:
Not sure to get what is the purpose of:

"If the client can display an error page to the end-user, it MUST generate the error page"

Reply:
I will remove this line.

Reply:
> I will remove this line.

Thanks, Tiru